packages/teleport/src/components/FormLogin/FormLogin.tsx

```diff
@@ -413,19 +418,23 @@ const Divider = () => (
     borderBottom={1}
     borderColor="text.placeholder"
     mx={5}
-    mt={2}
+    mt={5}
     mb={2}
   >
     <StyledOr>Or</StyledOr>
   </Flex>
 );
 
-const StyledPaswordlessBtn = styled(Box)`
-  cursor: pointer;
-  transition: all 0.3s;
+const StyledPaswordlessBtn = styled(ButtonText)`
```

Comment:
Great, this makes keyboard navigation possible.

One remaining issue is that after the sliding transition, for example going from step 1 to step 2, the focus for the page doesn't get reset. I press enter to submit the first step, but then:

* pressing enter does nothing (I expect it to submit the form)
* pressing tab focuses on the location bar, signaling that I tabbed past the last focusable element on the page

https://user-images.githubusercontent.com/27113/168096363-abbae751-67a8-465d-bc9f-dd9e356ea614.mov

Here I press tab after submitting the first step, expecting the tab to tab me through the form elements so that I can get to the device name field, but instead the browser focuses on the location bar.


https://user-images.githubusercontent.com/27113/168096593-a52d4e9b-86b6-490e-9b77-18705fed0d42.mov

Here I submit the first step and then press enter a couple of times, expecting the second step to be submitted, but nothing happens because there's no focus on the form. I then press tab expecting to be focused on the form, but instead the browser focuses on the location bar.

Off the top of my head I don't remember how to fix that though. 🤔 It's either through the `autofocus` attribute or some JS call that rewinds/resets the focus.

Reply:
@gzdunek See the last paragraph of my comment above, do you remember how to do this? IIRC you might have worked on something similar to that a few weeks ago.

Reply:
Yes, but in my case I had no input to focus, so I created a hidden one:
```typescript
const focusGrabber = (
  <input
    style={{
      opacity: 0,
      position: 'absolute',
      height: 0,
      zIndex: -1,
    }}
    autoFocus={true}
  />
);
```
but here we have an input, so `autoFocus` should work 🤔

Reply:
Thanks, I didn't notice this. I guess sometimes in Chrome, the `transitionend` event for `height` gets canceled (i wasn't sure why) but the fix was to also look for a `transitioncancel` for `height`. This will always autofocus on transition. Strangely this doesn't happen in firefox.

Reply:
I did a quick check and it looks like the reset form works now, but the problem is still present in the login form.

When I open the login form, then tab to "Other sign-in options" and press enter, the slide transition happens, I see the "Passwordless" button, but pressing enter does nothing and pressing tab focuses on the location bar.

Reply:
ah, i see what you mean, so on render/transition, focus on the first clickable button (i just assumed to just focus on input box regardless of position). i'll try to sneak this in my next PR

Reply:
> on render/transition, focus on the first clickable button

I'd say more like "on render/transition, focus on the first focusable [form control element](https://developer.mozilla.org/en-US/docs/Web/API/HTMLFormElement/elements).

In other words, despite having no "real" page transition here I'd like us to make the form behave as if we did transition into a page with a form that has a form control element with the `autofocus` attribute. I assume that this is what users would expect instead of the focus being somehow set at the last focusable element instead having tab change focus to the location bar.